Resources/Prototypes/_SpaceCats/Shaders/shaders.yml

```
@@ -1,4 +1,10 @@
 - type: shader
   id: NightVision
   kind: source
-  path: "/Textures/_SpaceCats/Shaders/nightvision.swsl"
\ No newline at end of file
+  path: "/Textures/_SpaceCats/Shaders/nightvision.swsl"
+
+- type: shader
+  id: FlickerHologram
+  kind: source
+  path: "/Textures/_SpaceCats/Shaders/flickerhologram.swsl"
+  light_mode: unshaded
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)